packages/react-cap-theme/src/components/Drawer/InlineDrawer.styles.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  makeStyles,
+  mergeClasses,
+  type InlineDrawerState,
+} from '@fluentui/react-components';
+
+export const useInlineDrawerStyles = makeStyles({
+  root: {},
```

Comment:
There is currently no design guidance for the inline option, but I created this file for potential future use. Please let me know if you would prefer it removed